src/components/PopupMenu/types.tsx

```diff
@@ -0,0 +1,45 @@
+import { PopupProps } from '../Popup'
+
+export type PopupMenuProps = {
+  /**
+   * The items to display in the menu
+   */
+  items: PopupMenuItem[]
+
```

Comment:
nit we should be consistent about newline spacing b/w prop fields

Reply:
Fixed!